bitnami/common/templates/_names.tpl

```diff
@@ -68,3 +68,14 @@ Create a fully qualified app name adding the installation's namespace.
 {{- define "common.names.fullname.namespace" -}}
 {{- printf "%s-%s" (include "common.names.fullname" .) (include "common.names.namespace" .) | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{/*
+ Create the name of the service account to use
+ */}}
+{{- define "common.names.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create -}}
```

Comment:
Note not always serviceAccount lies at the root of the chart values. Look at for example grafana-loki. This should be configured in a way that we allow setting different contexts, like we do in the image helpers

Reply:
the idea was a simple "define" which is useful for the most common case, and other cases could still do their `{{ define "$component.serviceAccountName" }}` 

I'll try your suggestion :)